src/encoded/types/biosample.py

```diff
@@ -1040,6 +1042,19 @@ def generate_summary_dictionary(
             phrase += ' (' + dict_of_phrases['age_display'] + ')'
         dict_of_phrases['sex_stage_age'] = phrase
 
+    if ('sample_type' in dict_of_phrases and
+        dict_of_phrases['sample_type'] == 'organoid'):
+        phrase = ''
+        if 'sample_term_name' in dict_of_phrases:
+            phrase += dict_of_phrases['sample_term_name']
+
+        if 'sample_type' in dict_of_phrases:
```

Comment:
I think you could omit this check since on lines 1045-1046 there is already a check for `sample_type` -- so it will always be true that dict_of_phrase['sample_type'] exists and == 'organoid'